### PR TITLE
dash/gulp-datagrid-lint

### PR DESCRIPTION
**Pull request by sebastianbochan**
Added separate linting to dashboards and datagrid.

### CODE REVIEW

File: tools/gulptasks/dashboards/dlint.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
```
Comment (karolkolodziej) [inline tools/gulptasks/dashboards/dlint.js:5]: We already have this tested in the `tools/gulptasks/dashboards/test.js` so there is no need to duplicate that.

Besides that, this task is not being used at all.

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by goransle — APPROVED** [review]

File: tools/gulptasks/dashboards/dlint.js (hunk 1 above)
Comment (sebastianbochan) [reply tools/gulptasks/dashboards/dlint.js:5]: Indeed! I was not aware of it, let's optimize it.

File: package.json [hunk 2]
```diff
@@ -131,14 +131,14 @@
...
     ],
-    "./ts/!(Dashboards)**/*.ts": [
+    "./ts/!(Dashboards|DataGrid)**/*.ts": [
       "npm run lint"
     ],
     "./ts/Dashboards/**/*.ts": [
-      "npm run lint-dashboards"
+      "cd ts/Dashboards && npx eslint"
     ],
-    "*.css": [
```
Comment (goransle) [inline package.json:140]: Please don't remove this 😬

**Review by goransle — CHANGES_REQUESTED** [review]
See comment

**Review by karolkolodziej — APPROVED** [review]

**Review by goransle — APPROVED** [review]

**Pull request merged**